### PR TITLE
the next 1.7.0 beta

### PR DESCRIPTION
**Pull request by shiftkey**
## Overview

 - [x] confirm we have everything for the beta merged into `development`
 - [x] get approval on release notes
 - [x] :shipit:

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+      "1.7.0-beta2": [
+        "[New] Rebase your current branch using a guided flow - #5953",
```
Comment (billygriffin) [inline changelog.json]: I'm not wedded to this but thoughts on being a bit more explanatory?
```suggestion
        "[New] Rebase another branch onto your current branch using a guided flow - #5953",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+        "[Fixed] Horizontal scroll bar visible in \"next steps\" section when switching branches - #7212",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
        "[Fixed] Horizontal scroll bar appears unnecessarily when switching branches - #7212",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+        "[Fixed] License templates do not end with newline character - #6999",
+        "[Fixed] Merge/Rebase conflicts banner do not clear when aborted outside Desktop - #7046",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
        "[Fixed] Merge/Rebase conflicts banners do not clear when aborting the operation outside Desktop - #7046",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+        "[Fixed] Missing tooltips for change indicators in the sidebar - #7174",
+        "[Fixed] Accessibility labels for icons would fail when multiple icons were visible at the same time - #7174",
```
Comment (billygriffin) [inline changelog.json]: Just changing this to match the tense of the other items.
```suggestion
        "[Fixed] Icon accessibility labels fail when multiple icons are visible at the same time - #7174",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,15 @@
...
+        "[Improved] Error handling of exceptions due to parsing API response - #7169",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
```

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+        "[Improved] Commit statuses are now updated more frequently - #7163"
```
Comment (billygriffin) [inline changelog.json]: From @niik's suggestion in the PR
```suggestion
        "[Improved] Pull requests load faster and PR build status updates automatically - #7163"
```

File: changelog.json (hunk 1 above)
Comment (shiftkey) [reply changelog.json]: I like this change but it feels the wrong way around - the current branch is rebased onto another branch of your choosing. What about this?

```suggestion
        "[New] Rebase your current branch onto another branch using a guided flow - #5953",
```

Comment (billygriffin) [reply changelog.json]: Sounds great!

**Pull request merged**